conf/solver/dds.yaml

```diff
@@ -1,17 +1,28 @@
 # @package _global_
 defaults:
-  - oc_base
-  - /prior: gauss_truncate
-  - /sde: vp_10
+  - basic_oc_base
+  - /prior: gauss
+  - /sde: 
   - /model@generative_ctrl: score
-  - /loss: reference_sde_lv_traj
+  - /loss: exponential_sde
   - _self_
 
 # Solver
 solver:
   _target_: sde_sampler.solver.oc.DDS
 
-# Model
-generative_ctrl:
-  clip_score: 10.
-  clip_model: 10.
+train_timesteps:
+  rescale_t: "cosine"
+  steps: 
+  end: 25.6 
+  dt: 0.05
+
+train_steps: 11000 
```

Comment:
We can remove lines `20-25` from the default config, to make settings more comparable.